ubuntu/Dockerfile

```diff
@@ -1,8 +1,105 @@
 FROM amd64/ubuntu:20.04
+
+# docker-to-linux
 LABEL com.iximiuz-project="docker-to-linux"
 RUN apt-get update -y
 RUN apt-get -y install \
   linux-image-virtual \
   systemd-sysv
 RUN echo "root:root" | chpasswd
 
+# docker-remote-desktop
+RUN sed -i -E 's/^# deb-src /deb-src /g' /etc/apt/sources.list \
```

Comment:
not really sure if we can support the multistage `FROM` stuff, so this is a copy and paste hackjob from https://github.com/scottyhardy/docker-remote-desktop/blob/master/Dockerfile